app/models/account.rb

```diff
@@ -37,6 +37,17 @@ def self.canonical_cname(cname)
     cname
   end
 
+  # Change the current active Account.
+  # Required until https://github.com/influitive/apartment/pull/307 is merged
+  # @param [String] cname the host name for the account
+  # @example
+  #   Account.use_account!('whatever.somewhere.io')
+  def self.use_account!(cname)
```

Comment:
I'm surprised this is a class method, with responsibility for both finding the account record and switching the connections. If you're happy with it in consuming code, though, :shipit:.